txnkv/transaction/2pc.go

```diff
@@ -1100,6 +1105,12 @@ const (
 )
 
 func (c *twoPhaseCommitter) cleanup(ctx context.Context) {
+	if c.store.IsClose() {
+		logutil.Logger(ctx).Error("twoPhaseCommitter fail to cleanup because the store exited",
```

Comment:
I think we can lower the log level to warning. It's not severe that some locks are not cleaned up on exit.

Reply:
Done